aries-site/src/pages/components/namevaluelist.mdx

```diff
@@ -155,7 +155,13 @@ In some cases, NameValuePairs may be used as read-only display for text inputs t
 
 ### Content Truncation
 
-When a value content exceeds 192 characters, truncate content and include a Button to expand to see more.
+When presenting a list of items, the list may be truncated and 
+accompanied by a Button to expand the list, displaying all items.
```

Comment:
When presenting a value as a list of items, the list may be truncated and accompanied by a Button to expand it, displaying all items.

Reply:
Good call. Done.